lib/transformer.dart

```diff
@@ -258,21 +256,23 @@ class _BootstrapFileBuilder {
               '(possibly transitive).');
         }
       }
+
       readSuperClassAnnotations(clazz.supertype);
       _readAnnotations(clazz);
     }
   }
 
   bool _readAnnotations(Element element) {
     var found = false;
-    if (element.metadata.isEmpty) return found;
+    // analyzer 0.29 doesn't allow this optimization :
+    //if (element.metadata.isEmpty) return found;
```

Comment:
Well, I could try creating a project with many many imports ... I'll try that way and see if that will reproduce the problem.